client/components/quizSelection/TopicSelection.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { TopicSelectionContainer, TopicSelectionList, TopicSelectionItem } from './styles';
+import { PrimaryButton, PrimaryButtonSmall, TextBodySmall } from '../shared/styles';
```

Comment:
Not that it matters, but I would probably put shared styles before custom ones (just to show order of importance)

Reply:
Fixed in latest push, tried to correct this anywhere I saw it.